js/dropdown.js

```diff
@@ -44,7 +44,7 @@
 
       if (!$parent.hasClass('open')) return
 
-      if (e && e.type == 'click' && /input|textarea/i.test(e.target.tagName) && $.contains($parent[0], e.target)) return
+      if (e && e.type == 'click' && (/input|select|textarea/i.test(e.target.tagName) || /checkbox|radio/i.test(e.target.className)) && $.contains($parent[0], e.target)) return
```

Comment:
Seems like this should be `type` instead of `className`. Folks might be using vanilla non-Bootstrap checkboxes.